chatbot/src/main/java/chatbot/core/handlers/eliza/ElizaHandler.java

```diff
@@ -13,18 +16,27 @@
 	public ElizaHandler() {
 
 	}
-
-	public String search(IncomingRequest request) throws JsonProcessingException, IOException {
+	public Response createNewResponse(String response)
+	{
+		Response obj = new Response();
+		obj.setContent(response);
+		obj.setTitle("");
+		return obj;
+	}
+	public ResponseList search(IncomingRequest request) throws JsonProcessingException, IOException {
 		try {
+			ResponseList responselist = new ResponseList();
 			String query = request.getRequestContent()
 			                      .get(0)
 			                      .getText();
 			ElizaMain eliza = new ElizaMain();
 			eliza.readScript(true, "src/main/resources/eliza/script");
 			String response = eliza.processInput(query);
 			log.info("search:Got Response , response= " + response);
-			String responseJSON = "[{ \"comment\":\"" + response + "\"}]";
-			return responseJSON;
+			responselist.setMessageType(MessageType.PLAIN_TEXT);
+			responselist.addMessage(createNewResponse(response)); // Add Response to Final String
+			//String responseJSON = "[{ \"comment\":\"" + response + "\"}]";
+			return responselist;
 		}
 		/*catch (JsonProcessingException e) {
```

Comment:
The search method throws JsonProcessingException and IOException but here it is commented out..Is it supposed to be added later?

Reply:
@sepidetari  They dont get thrown by this class so it is not required to handle those 2 Exceptions.